context_test.go

```diff
@@ -0,0 +1,34 @@
+package gb
+
+import (
+	"runtime/debug"
+	"strings"
+	"testing"
+)
+
+func testImportCycle(pkg string, t *testing.T) {
```

Comment:
Oh, I didn't even see that. I'll merge this as is, it'll get messy to hammer it into the general test.